src/Compilers/CSharp/Test/Semantic/Semantics/LookupTests.cs

```diff
@@ -1443,7 +1443,7 @@ public static void Main()
 }
 ";
             // Get the list of LookupSymbols at the location of the CSharpSyntaxNode
-            var actual_lookupSymbols = GetLookupSymbols(usings.ToString() + source, isScript: false);
+            var actual_lookupSymbols = GetLookupSymbols(string.Join("\r\n", usings) + source, isScript: false);
```

Comment:
>"\r\n" [](http://example.com/codeflow?start=68&length=6)

The motivation for this change is not obvious.

Reply:
The test wasn't properly written, and started to fail with my changes.

Now you already fixed that in https://github.com/dotnet/roslyn/commit/9d5f1ace8506f3d0d0cef8cba1e472a9896c2258

So there will be no change here once I merge latest main and fix the conflict.